### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by nicholasday**

### CODE REVIEW

**Comment by Dafondo** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+# Usage
+
+Log in to Devpost, after installing the extension. Go to Account -> Manage Hackathons -> Submissions. Once you're on the submissions page, click the DevGET button. The results from the scraping should be in the console. (Soon to be downloadable and normalized in CSV files)
+
+# Installation
+
+1. Clone the repo
+
+2. Peruse Chrome's extension [getting started tutorial](https://developer.chrome.com/extensions/getstarted) at your leisure
+  * We will walk through the gist of it in the following steps
```
Comment (Dafondo) [inline README.md]: **Hey** this indent _is_`n't` displaying, please fix. 🥇 :)
@nicholasday 

- [ ] TODO: Replace with actual todo
- [ ] Check this box

**Review by Dafondo — CHANGES_REQUESTED** [review]
See comment. pls 2 halp. no gud at compter.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+
+3. In Chrome, go to `chrome://extensions`
+
+4. Toggle the `Developer Mode` switch on the page
+
+5. Click `LOAD UNPACKED`
```
Comment (yicheng-w) [inline README.md]: Unable to find `LOAD UNPACKED`, did you mean `Pack Extension`? Please clarify.

unclear

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+
+6. Select the directory of this repo that you cloned earlier
+
+# Contribution
```
Comment (yicheng-w) [inline README.md]: Maybe rename to `Files`, Contribution sounds weird... it's not that big a open source repo anyway.

**Review by yicheng-w — CHANGES_REQUESTED** [review]

**Review by Dafondo — APPROVED** [review]

**Review by yicheng-w — APPROVED** [review]

**Comment by yicheng-w** [comment]
:+1:

**Pull request merged**